src/test/resources/application.yml

```diff
@@ -2,7 +2,7 @@
 logging:
   level:
     root: "WARN"
-    org.pih: "DEBUG"
+    org.pih: "INFO"
```

Comment:
Moved the default logging to INFO.  We can (and should) adjust logging through Puppet as needed to further tweak what is helpful to view in the logs.  For now, I'm going for less verbose overall.